src/pages/NotFoundStyling.ts

```diff
@@ -0,0 +1,22 @@
+import { css } from "@emotion/core";
```

Comment:
Since notFoundStyling is used in multiple files, maybe rename it to `errorPageStyling`